modules/ROOT/pages/acl-policy.adoc

```diff
@@ -3,25 +3,25 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The IP Whitelist policy allows you to create an explicit list of IP addresses that can access your deployed endpoints. IP addresses that aren't on this white list are rejected. 
+The IP Whitelist policy allows you to create an explicit list of IP addresses that can access your deployed endpoints. IP addresses that aren't on this list are rejected.
```

Comment:
Why didn't it get changed here? Is this in the UI?

Reply:
Even if this is the UI, can it be rewritten to something like 

You can create an explicit list of IP addresses that can access your deployed endpoints. IP addresses that aren't on this list are rejected.

Reply:
IP Whitelist is a policy, I just tried to remove anything that wasn't directly referring to UI or the policy as a proper noun. It's kind of weird, some things get flagged as changeable and some don't. This probably got flagged since it's not spelt correct, felt it could be dropped to list because it's not UI and the white is implied clearly.